### PR TITLE
item storage tweaks

### PR DESCRIPTION
**Pull request by 567Turtle**
## About The Pull Request

microscopic tweak which adds a few things to the roster of items that can be held in the corpsmans white vest

## Why It's Good For The Game

who needs a pill bottle when they can have a whistle in their vest? also I always wanna heal my robotic squadmates as a corpsman but never get the chance to because I can't carry a soldering tool

## Changelog
:cl:
balance: you can now hold a whistle and a soldering tool in the white webbing vest, as well as an injector in your boots
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
Whistle is whatever, but defib is a normal sized item, where as white vest only holds small items.

**Comment by 567Turtle** [comment]
> Whistle is whatever, but defib is a normal sized item, where as white vest only holds small items.
> 
> Also why are you editing an old changelog?

The defib is pretty cumbersome and annoying to carry around, I thought it'd be fine to suspend our disbelief for a second and attach the defib to the outside of the webbing or something of the sort.

Didn't mean to edit that changeling, was just going through and fixing a common grammar error I saw, didn't pay much mind to where that error was, shouldn't cause any problems though.

**Comment by Xsdew** [comment]
The defib being cumbersome to carry around is intentional as a balance decision. For example, the Life Saver Rig carriers more supplies but is more limited as to what can fit. The Medical Rig carriers less but can carry items the Life Saver Rig cannot including the defib. The purpose of the White Webbing is to carry +1 item compared to the Black / Brown but limited to certain Medical Supplies.

**Comment by 567Turtle** [comment]
Yeah now that you explain it, makes sense

**Comment by 567Turtle** [comment]
I still want my whistle

**Comment by Losenis** [comment]
Defib in white vest only enables loadouts without backpack since that's where most put the defib, and going without backpack has a way bigger drawback of not having the ability to recharge the defib once it runs out of battery. If anything it'd enable more variety in corpsman loadouts without having the backpack be an absolutely required item, since medrig is a meme that only non-corpsmen benefit in taking due to not needing a quadrillion kits and splints that the lifesaver belt's additional storage allows you to take.

**Comment by 567Turtle** [comment]
No longer does the defib fit in the white vest, I see why that was a thing now, makes sense. Now the only thing this PR adds to the white vest is the whistle and the sodering tool

**Comment by 567Turtle** [comment]
also went ahead and made it so you can put an auto injector in your boots

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Missing trailing commas

Also no to the autoinjector because thats storage creep

**Comment by 567Turtle** [comment]
> Missing trailing commas
> 
> Also no to the autoinjector because thats storage creep

Didn't know you needed trailing commas if it was the last item in a list, I'll fix it in a bit

File: html/changelogs/changelog2017.html [hunk 1]
```diff
@@ -2368,7 +2368,7 @@ <h2 class='date'>22 January 2017</h2>
 	<h3 class='author'>Rahlzel updated:</h3>
 	<ul class='changes bgimages16'>
 		<li class='rscdel'>Cloning system removed!</li>
-		<li class='tweak'>Defibrilators have been buffed significantly. Players dead within 5 minutes can be revived as long as the defibrilator has enough charges, and as long as the player didn't die from suicide, chestbursting, head removal or extreme damage. Defibrilators can be charged using the same rechargers that are used for tasers and stun batons.</li>
+		<li class='tweak'>Defibrilators have been buffed significantly. Players dead within 5 minutes can be revived as long as the defibrillator has enough charges, and as long as the player didn't die from suicide, chestbursting, head removal or extreme damage. Defibrilators can be charged using the same rechargers that are used for tasers and stun batons.</li>
```
Comment (BraveMole) [inline html/changelogs/changelog2017.html:2371]: Do not touch old changelog please. Typos do not matter there

**Review by BraveMole — APPROVED** [review]

File: html/changelogs/changelog2017.html (hunk 1 above)
Comment (567Turtle) [reply html/changelogs/changelog2017.html:2371]: Alright sure yeah, thought I reverted those changes already, I'll fix it in a bit

**Pull request merged**